packages/contentstack-bootstrap/src/bootstrap/utils.ts

```diff
@@ -34,6 +34,40 @@ export const setupEnvironments = async (
     .environment()
     .query()
     .find();
+
+  //create management token if not present
+  if(!managementToken){
+    const managementBody = {
+      "token":{
+          "name":"sample app",
+          "description":"This is a sample management token.",
+          "scope":[
+              {
+                  "module":"content_type",
+                  "acl":{
+                      "read":true,
+                      "write":true
+                  }
+              },
+              {
+                  "module":"branch",
+                  "branches":[
+                      "main"
+                  ],
+                  "acl":{
+                      "read":true
+                  }
+              }
+          ],
+          "expires_on": "3000-01-01",
+          "is_email_notification_enabled":false
+      }
+  }
+  managementTokenResult = await managementAPIClient
+    .stack({ api_key: api_key })
+    .managementToken()
+    .create(managementBody);
+  }
```

Comment:
@sunil-lakshman , Is the creation of a management token required? We can throw errors in place of creating management tokens. We're attempting to create management tokens without user permission, and I believe this is going to be a major problem.
cc: @netrajpatel @shafeeqd959

Reply:
@aman19K for other apps we are creating delivery token internally, Bootstrap flow should setup all the required environment values internally.

@sunil-lakshman Correct if I am wrong, My understanding here is that we need to have management token here to make this app working.

Reply:
@shafeeqd959 Management token required for compass app

Reply:
@shafeeqd959 , creating management token without user consent. isn't it going to be problem? May be some prompt for confirmation so that they'll aware about it.

Reply:
@aman19K,  what about other apps where we create delivery tokens ?

Reply:
This point is valid,  we have to make a decision that, is it just required here or in other apps as well, if needed we can have one ticket to make that change.